### PR TITLE
Deprecate FormFooterItem

### PR DESCRIPTION
**Pull request by descorp**
### Changes

* remove footer note from SEPA
* deprecate FormFooterItem, FormFooterView and FormFooterStyle
* rename ComponentFormItem into FormComponentItem
* use FormButtonItem instead of FormFooterItem on all components

### CODE REVIEW

File: Adyen/UI/Form/Items/Basic Items/FormContainerItem.swift [hunk 1]
```diff
@@ -7,9 +7,14 @@
 import Foundation
 
 /// Simple form item to wrap another item and provide a margin around it.
-public class ContainerFormItem: FormItem {
+public class FormContainerItem: FormItem {
```
Comment (mohammedDehairy) [inline Adyen/UI/Form/Items/Basic Items/FormContainerItem.swift:10]: this can hidden from docs.

File: Adyen/Components/BLIK/BLIKComponent.swift [hunk 2]
```diff
@@ -84,22 +82,22 @@ public final class BLIKComponent: PaymentComponent, PresentableComponent, Locali
     }()
 
     /// The footer item.
-    internal lazy var footerItem: FormFooterItem = {
-        let footerItem = FormFooterItem(style: style.footer)
-        footerItem.submitButtonTitle = ADYLocalizedSubmitButtonTitle(with: payment?.amount, localizationParameters)
-        footerItem.submitButtonSelectionHandler = { [weak self] in
+    internal lazy var button: FormButtonItem = {
+        let item = FormButtonItem(style: style.mainButtonItem)
+        item.identifier = ViewIdentifierBuilder.build(scopeInstance: self, postfix: "footerItem")
```
Comment (mohammedDehairy) [inline Adyen/Components/BLIK/BLIKComponent.swift:87]: the identifier postfix should be `buttonItem` not `footerItem `

File: AdyenTests/Adyen Tests/Components/MB Way/MBWayComponentTests.swift [hunk 3]
```diff
@@ -84,8 +84,8 @@ class MBWayComponentTests: XCTestCase {
             let phoneNumberViewTitleLabel: UILabel? = sut.viewController.view.findView(with: "Adyen.MBWayComponent.phoneNumberItem.titleLabel")
             let phoneNumberViewTextField: UITextField? = sut.viewController.view.findView(with: "Adyen.MBWayComponent.phoneNumberItem.textField")
 
-            let footerItemViewButton: UIControl? = sut.viewController.view.findView(with: "Adyen.MBWayComponent.footerItem.submitButton")
-            let footerItemViewButtonTitle: UILabel? = sut.viewController.view.findView(with: "Adyen.MBWayComponent.footerItem.submitButton.titleLabel")
+            let footerItemViewButton: UIControl? = sut.viewController.view.findView(with: "Adyen.MBWayComponent.footerItem.button")
+            let footerItemViewButtonTitle: UILabel? = sut.viewController.view.findView(with: "Adyen.MBWayComponent.footerItem.button.titleLabel")
```
Comment (mohammedDehairy) [inline AdyenTests/Adyen Tests/Components/MB Way/MBWayComponentTests.swift:88]: why is `footerItem` is still in the identifier?

Comment (descorp) [reply AdyenTests/Adyen Tests/Components/MB Way/MBWayComponentTests.swift]: `payButtonItem` now

File: Adyen/Components/BLIK/BLIKComponent.swift (hunk 2 above)
Comment (descorp) [reply Adyen/Components/BLIK/BLIKComponent.swift]: `payButtonItem` now

File: Adyen/UI/Form/Items/Basic Items/FormContainerItem.swift (hunk 1 above)
Comment (descorp) [reply Adyen/UI/Form/Items/Basic Items/FormContainerItem.swift:11]: done

**Review by mohammedDehairy — APPROVED** [review]

**Pull request merged**